Gemfile

```diff
@@ -1,3 +1,5 @@
 source "http://rubygems.org"
 
-gemspec
\ No newline at end of file
+gemspec
+
+gem 'activesupport', ENV.fetch('RAILS_VERSION', '>= 6.1.7')
```

Comment:
Is it possible to add this to the gemspec?

Reply:
I've moved this out to dedicated gemfiles, one for each version of Rails used in the CI build.